scanpipe/filters.py

```diff
@@ -349,9 +349,13 @@ def __init__(self, data=None, *args, **kwargs):
 
         # Default filtering by "Active" projects.
         if not data or data.get("is_archived", "") == "":
-            self.queryset = self.queryset.filter(is_archived=False)
+            self.queryset = self.queryset.filter(
+                is_archived=False, is_marked_for_deletion=False
+            )
```

Comment:
It would make sense to add a new `active()` QuerySet method to avoid code duplication.